sklearn/ensemble/_base.py

```diff
@@ -148,7 +149,7 @@ def _make_estimator(self, append=True, random_state=None):
         sub-estimators.
         """
         estimator = clone(self.base_estimator_)
-        estimator.set_params(**{p: getattr(self, p)
+        estimator.set_params(**{p: copy.deepcopy(getattr(self, p))
```

Comment:
```suggestion
        # make a deepcopy in case one of the base estimator has a mutable
        # parameter that might be shared and modified during parallel fitting
        estimator.set_params(**{p: copy.deepcopy(getattr(self, p))
```